features/step_definitions/viewing_browse_steps.rb

```diff
@@ -115,28 +113,25 @@ def top_level_browse_pages
 end
 
 def add_browse_pages
-  content_store_has_item '/browse', { links: {
+  content_store_has_item '/browse', links: {
     top_level_browse_pages: top_level_browse_pages
-  }}
+  }
 end
 
-def add_first_level_browse_pages(child_pages: second_level_browse_pages, order_type: order_type)
```

Comment:
This method is being used on line 11 and 45

Inside this method, `second_level_browse_pages` becomes `child_pages` (used in line: 125 of this commit) and `order_type`is in use on line 128